perftracker/templates/comparison_id.html

```diff
@@ -227,9 +227,9 @@ <h4>Job: {{ job }}</h4>
         const selector_id = `${s_id}_${property}_selector`;
```

Comment:
from line above: I missed that in previous review: "graph_properties" - "graph" is rather incorrect terminology